examples/housing_market_research/investor_analytics/portfolio_analysis.ipynb

```diff
@@ -0,0 +1,864 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Environment setup\n",
+    "import os\n",
+    "from io import BytesIO\n",
+    "import pandas as pd\n",
+    "import matplotlib.pyplot as plt\n",
+    "import matplotlib.dates as mdates\n",
+    "import plotly\n",
+    "from PIL import Image\n",
+    "import requests as re\n",
+    "import numpy as np\n",
+    "import pandas as pd\n",
+    "import plotly.express as px\n",
+    "from datetime import datetime\n",
+    "import plotly.graph_objects as go\n",
+    "from parcllabs import ParclLabsClient\n",
+    "from parcllabs.beta.charting.utils import create_labs_logo_dict\n",
+    "from parcllabs.beta.charting.styling import default_style_config as style_config\n",
+    "from parcllabs import ParclLabsClient\n",
+    "\n",
+    "# Create a Parcl Labs client\n",
+    "client = ParclLabsClient(\n",
+    "    api_key=os.environ.get('PARCL_LABS_API_KEY', \"<your Parcl Labs API key if not set as environment variable>\"), \n",
+    "    limit=10 # set default limit\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We are going to retrieve the concentration of single-family homes by investor portfolio size, which can be accessed using the `portfolio_metrics.sf_housing_stock_ownership` method. Alternatively, you can access this data via the API using this [endpoint](https://docs.parcllabs.com/reference/sf_housing_stock_ownership_v1_portfolio_metrics__parcl_id__sf_housing_stock_ownership_get-1). This analysis will help us understand the composition of the SFR market in the Milwaukee metro area. We will examine investor concentration as of July 2024. The `portfolio_metrics.sf_housing_stock_ownership` method accepts `parcl_ids`, `start_date`, and `end_date` as parameters, so we need to define them in a dictionary that we can pass to the client."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Focus on two of the largest investors\n",
+    "housing_stock_ownership = {\n",
+    "    'parcl_ids': [2900134],  # Replace with appropriate Parcl IDs if needed\n",
+    "    'start_date': '2024-07-01',  # Set the start date for the data retrieval, data starts from 2024-03-01\n",
+    "    'end_date': '2024-07-31',    # Set the end date for the data retrieval\n",
+    "    'limit': 100,                # Limit the number of records retrieved\n",
```

Comment:
In comments you can show that users can always gett the most recent value using: 

```python
housing_stock_ownership = {
    'parcl_ids': [2900134],  # Replace with appropriate Parcl IDs if needed
    'limit': 1,                # Limit the number of records retrieved
}
```

Reply:
Added to notebook